frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -83,10 +84,26 @@ export const BrowseModelsBody = ({
       actualModelFilters,
       availableModelFilters,
     );
-    return filteredModels;
+    return filteredModels.slice(0, 100);
```

Comment:
this was just to make the page load in a reasonable amount of time in my local environment 😂